Makefile.olm

```diff
@@ -0,0 +1,169 @@
+	   --inplace  bundle/manifests/k8gb.clusterserviceversion.yaml
+	@echo "$(YELLOW)Display names and descriptions have been added$(NC)\n"
+
+.PHONY: bundle-fix-dockerfile
+bundle-fix-dockerfile: ## Removes 1 line from bundle dockerfile (scorecard entry)
+	sed -i "/^COPY bundle\/tests\/scorecard/d" bundle.Dockerfile
+	@echo "$(YELLOW)Scorecard entry has been removed from the bundle.Dockerfile$(NC)\n"
+
+.PHONY: bundle-remove-local
+bundle-remove-local: ## Removes ./bundle and ./bundle.Dockerfile (these are generated by make bundle-generate)
+	-rm -Rf ./bundle
+	-rm -Rf ./bundle.Dockerfile
+	@echo "$(YELLOW)Local ./bundle directory has been removed$(NC)\n"
+
+.PHONY: bundle-full
+bundle-full: deployment-blueprint bundle-remove-local bundle-generate bundle-fix-dockerfile bundle-add-examples bundle-add-crd-info bundle-validate bundle-image-build ## Prepares the container image w/ the bundle including the necessary intermediate steps
+	@echo "\n\n$(YELLOW)Full container image ($(BUNDLE_IMAGE_REPO)/$(BUNDLE_IMAGE_NAME):$(BUNDLE_VERSION)) w/ bundle has been built$(NC)\n"
+	@echo "Continue with make bundle-image-push\n"
+
+.PHONY: bundle-validate
+bundle-validate: ## Validate the CSV file
+	@echo "$(CYAN)Validating bundle..$(NC)\n"
+	$(call operator-sdk,bundle,validate,./bundle,--optional-values=index-path=bundle.Dockerfile)
+	@echo "$(YELLOW)Bundle is OK!$(NC)\n"
+
+.PHONY: bundle-image-build
+bundle-image-build: ## Builds the container image w/ bundle metadata
+	@echo "$(CYAN)Building bundle container image..$(NC)\n"
+	docker build -f bundle.Dockerfile -t $(BUNDLE_IMAGE_REPO)/$(BUNDLE_IMAGE_NAME):$(BUNDLE_VERSION) .
+
+.PHONY: bundle-image-push
+bundle-image-push: ## Push the bundle image to the pre-defined image registry
+	docker push $(BUNDLE_IMAGE_REPO)/$(BUNDLE_IMAGE_NAME):$(BUNDLE_VERSION)
+	@echo "\nTo deploy the new bundle do: make bundle-deploy\n"
+
+# this assumes the OpenShift is up and running with k8gbtest namespace created
+.PHONY: bundle-deploy
+bundle-deploy: ## Deploys the bundle image into OpenShift's catalog, this will also deploy the operator
+	$(call operator-sdk,run,bundle,-n,$(PREFERRED_NS),$(BUNDLE_IMAGE_REPO)/$(BUNDLE_IMAGE_NAME):$(BUNDLE_VERSION))
+
+.PHONY: cleanup
+cleanup: ## Deletes the k8gb operator from the OpenShift catalog
+	$(call operator-sdk,cleanup,k8gb)
+
+.PHONY: help
+help: ## Show this help
+	@egrep -h '\s##\s' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
+
+
+###############################
+#		FUNCTIONS
+###############################
+
+define operator-sdk
+	@which operator-sdk > /dev/null || { \
+		echo "$(YELLOW)Downloading operator-sdk..$(NC)" ; \
+		curl -sLo $(GOBIN)/operator-sdk $(OPERATOR_SDK_DL_URL)/operator-sdk_$(OS)_$(ARCH) ; \
+		chmod +x $(GOBIN)/operator-sdk ; \
+	}
+	$(GOBIN)/operator-sdk $1 $2 $3 $4 $5 $6 $7 $8 $9 $(10)
```

Comment:
can't we just `$(GOBIN)/operator-sdk $1` and than 
```Makefile
$(call operator-sdk,cleanup k8gb)
$(call operator-sdk,run bundle -n $(PREFERRED_NS) $(BUNDLE_IMAGE_REPO)/$(BUNDLE_IMAGE_NAME):$(BUNDLE_VERSION)
...
```

Reply:
I was looking for something like `$@` b/c I hate that line with `$1 $2 $3 $4 $5 $6 ..` I've tried your approach and it works so I am changing it